ambuda/seed/dcs.py

```diff
@@ -6,7 +6,6 @@
 import ambuda.database as db
 from ambuda.seed.utils.itihasa_utils import create_db
 
-
```

Comment:
is isort8 removing these lines?

Reply:
Yes, isort likes only one line between imports and subsequent code
 ```
$ isort --diff --check ~/src/ambuda-kvic/ambuda/seed/dcs.py 
ERROR: /Users/kvic/src/ambuda-kvic/ambuda/seed/dcs.py Imports are incorrectly sorted and/or formatted.
--- /Users/kvic/src/ambuda-kvic/ambuda/seed/dcs.py:before	2022-09-06 22:02:29.609973
+++ /Users/kvic/src/ambuda-kvic/ambuda/seed/dcs.py:after	2022-09-06 22:02:40.348053
@@ -5,7 +5,6 @@
 
 import ambuda.database as db
 from ambuda.seed.utils.itihasa_utils import create_db
-
 
 REPO = "https://github.com/ambuda-org/dcs.git"
 PROJECT_DIR = Path(__file__).resolve().parents[2]